src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -382,6 +382,20 @@ private EtwEventSource()
             this.WriteEvent(223, TaskHub, AppName, SlotName, Details, ExtensionVersion);
         }
 
+        [Event(224, Level = EventLevel.Warning)]
```

Comment:
We are probably good, but as a part of ANT87 we should consider adding more of a buffer to these events.

Reply:
Agreed.